src/libcore/python/object.cpp

```diff
@@ -30,7 +30,8 @@ class PyTraversalCallback : public TraversalCallback {
 };
 
 MTS_PY_EXPORT(Object) {
-    py::class_<Class>(m, "Class", D(Class));
+    py::class_<Class>(m, "Class", D(Class))
+        .def_method(Class, parent, py::return_value_policy::reference);
```

Comment:
Let's not expose this low-level logic to Python. The `PluginManager.create(..)` should be able to handle this.